packages/suite/src/actions/wallet/graphActions.ts

```diff
@@ -21,7 +21,7 @@ import {
     enhanceBlockchainAccountHistory,
     accountGraphDataFilterFn,
     deviceGraphDataFilterFn,
-} from '../../utils/wallet/graphUtils';
+} from 'src/utils/wallet/graph';
 import { selectLocalCurrency } from '../../reducers/wallet/settingsReducer';
```

Comment:
```suggestion
import { selectLocalCurrency } from 'src/reducers/wallet/settingsReducer';
```

and above `../../types/wallet/graph`

Reply:
All imports from above and maybe some more fixed in 3f441332fca5581de32856deb5046442c3e4973d